Socket/client/chatapp/src/Component/Common/ChatLists/Channels/ChatChannels/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+// @ts-ignore
+import { ArrowDown2TwoToneIcon, ArrowUp2TwoToneIcon } from "@iconbox/iconly";
+// @ts-ignore
+import { StarFillIcon } from "@iconbox/eva";
+
+import Group from "../../../Group";
+
+import { StyledChatChannelsWrapper, StyledHeaderWrapper } from "./style";
+
+interface Props {
+  socket: {
+    on: Function;
+  };
```

Comment:
Use `Socket` type from `socket.io` client: https://socket.io/docs/v4/typescript/#types-for-the-client